code/game/turfs/simulated/walls.dm

```diff
@@ -370,15 +370,34 @@
 		return TRUE
 	return FALSE
 
-/turf/simulated/wall/proc/try_decon(obj/item/I, mob/user, params)
+/turf/simulated/wall/proc/try_decon(obj/item/I, mob/living/user, params)
```

Comment:
Why specify `/living` here?

Reply:
Echoing this. What is the point of `living` here.

*Do not take this out of context*

Reply:
Removed